inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/README.md

```diff
@@ -16,8 +16,14 @@ Outcome of a build is a `subgrpahsDumper` binary located in building artifacts f
 
 ## Running
 The tool takes two command line parameters:    
-* `--input_folders` - comma separated list for recursive search for models to analyse
-* `--output_folder` - path to the output folder which will be used for extracted subgrpahs serialization in IR format
+* `--input_folders` - Required. Comma separated paths to the input folders with IRs
+* `--output_folder` - Required. Path to the output folders where to serialize IRs
+* `--path_regex` - Optional. regular expression to be applied in input folders recursive discovery
+* `--constants_size_threshold` - Optional. Maximum size of constant in megabytes to be serialized.
+                                 If constant size exceeds specified number it will be replaced
```

Comment:
If constant size exceeds specified number it will be replaced with parameter and meta information... --->   If constant size exceeds specified number, it will be replaced with parameter, and meta information...